README.md

```diff
@@ -84,60 +116,70 @@ $ yarn dev
 
 > Finally, you can go to [127.0.0.1:5000](http://127.0.0.1:5000/) and view the admissions page.
 
-To create an admission, first, go to [http://127.0.0.1:5000/login/lego/](http://127.0.0.1:5000/login/lego/) to authorize. Then, click "Administrer opptak" and create an admission. Phew, now you are ready to start developing!
+**NB: The project has to be accessed through 127.0.0.1, and NOT localhost.** This is because accessing both LEGO and admissions from the same hostname creates a conflict some session storage, so the login will not work.
 
-## Requirements
+To create an admission, first, open [127.0.0.1:5000](http://127.0.0.1:5000/) and click the "Logg inn" button at the bottom of the page to authorize as a user with [permission to create admissions](#permissions). Then, click "Administrer opptak" and create an admission. Phew, now you are ready to start developing!
 
-We use the package [pip-tools](https://github.com/jazzband/pip-tools) to organize our requirements.
+&nbsp;
 
-### Adding a new requirement
+## Creating admissions
 
-Add the new package to either `base.in`, `development.in` or
-`production.in` depending on best fit. Then generate a new requirements
-file by running
+### GUI
 
-We use [pip-tools](https://github.com/jazzband/pip-tools) to make the requirement files easy to understand and maintain.
-Run the following custom command to update `development.txt` and `production.txt`.
+The simplest way to create an admission is through the GUI.
 
-```sh
-$ poetry run python manage.py compile_requirements
-```
+1. Navigate to [127.0.0.1:5000](http://127.0.0.1:5000/)
+2. Log in as a user with permission to create admissions.
+3. Click "Administrer opptak" at the bottom of the screen
+4. Success
+
+### Shell
 
-Or the manual way
+Currently when running
 
 ```sh
-$ pip-compile requirements/development.in > requirements/development.txt
+# Create a custom admission for development
+$ poetry run python manage.py create_admission
 ```
 
-To sync your virtual environment with the requirements file, do
+you create an admission connected to all groups, if they exist (they are generated the first time you log in). To connect it to a group, you can either do it through the GUI, or create it in the shell.  
```

Comment:
```suggestion
You create an admission connected to all groups, if they exist (they are generated the first time you log in). To connect it to a group, you can either do it through the GUI, or create it in the shell.  
```

Reply:
This one was not capitalized, as it is meant to be read as a continuation of the sentence before. Just got a bit separated because of the code in between. Thus I'd prefer to keep it as is